internal/cli/atlas/clusters/update.go

```diff
@@ -116,18 +129,10 @@ func UpdateBuilder() *cobra.Command {
 		fs: afero.NewOsFs(),
 	}
 	cmd := &cobra.Command{
-		Use:   "update [clusterName]",
-		Short: "Update a MongoDB cluster.",
-		Example: `
-  Update tier for a cluster
-  $ mongocli atlas cluster update <clusterName> --projectId <projectId> --tier M50
-
-  Update disk size for a cluster
-  $ mongocli atlas cluster update <clusterName> --projectId <projectId> --diskSizeGB 20
-
-  Update MongoDB version for a cluster
-  $ mongocli atlas cluster update <clusterName> --projectId <projectId> --mdbVersion 4.2`,
-		Args: require.MaximumNArgs(1),
+		Use:     "update [clusterName]",
+		Short:   "Update a MongoDB cluster.",
+		Example: fmt.Sprintf(exampleUpdate, exampleCmd, exampleCmd, exampleCmd),
```

Comment:
nit: this made me learn something https://stackoverflow.com/questions/37001449/replace-all-variables-in-sprintf-with-same-variable and maybe we can try one of the options here? e.g. `s := fmt.Sprintf("%[1]v in %[1]v is %[1]v", val)
`

Reply:
Thank you for sharing! Updated